packages/editor-ui/src/components/Node.vue

```diff
@@ -161,6 +180,42 @@ export default mixins(externalHooks, nodeBase, nodeHelpers, workflowHelpers).ext
 				this.setSubtitle();
 			}
 		},
+		nodePosition: {
+			handler() {
+				if (this.showWebhookNodeTooltip) {
+					this.showWebhookNodeTooltip = false;
+					setTimeout(() => {
+						this.showWebhookNodeTooltip = true;
+					}, 200);
+				}
+			},
+			immediate: true,
+			deep: true,
+		},
+		nodeOffsetPosition: {
+			handler() {
+				if (this.showWebhookNodeTooltip) {
+					this.showWebhookNodeTooltip = false;
+					setTimeout(() => {
+						this.showWebhookNodeTooltip = true;
+					}, 200);
+				}
+			},
+			immediate: false,
+			deep: true,
+		},
+		workflowRunning: {
+			handler(isWorkflowRunning) {
+				if (isWorkflowRunning && this.isTriggerNode && this.isSingleActiveTriggerNode && !this.isTriggerNodeTooltipEmpty) {
+					setTimeout(() => {
+						this.showWebhookNodeTooltip = isWorkflowRunning;
```

Comment:
this is still buggy.. I think you need to check here again that it's still running
<img width="607" alt="Screen Shot 2021-11-17 at 9 30 05 AM" src="https://user-images.githubusercontent.com/4711238/142163755-f6c3fe75-5aba-4521-859d-3d72f0192adc.png">
to reproduce:
1. execute workflow
2. pause before tooltip shows

Reply:
if node errors out, still shows tooltip
<img width="628" alt="Screen Shot 2021-11-17 at 9 38 08 AM" src="https://user-images.githubusercontent.com/4711238/142165894-ffdb2920-bcee-44a6-81c7-d9e67344a96f.png">

Reply:
same thing if node runs successfully before timeout
<img width="515" alt="Screen Shot 2021-11-17 at 9 40 35 AM" src="https://user-images.githubusercontent.com/4711238/142166306-0d8cd4dc-0bc7-46e1-beb2-51be7847d24d.png">

Reply:
This three scenarios are fixed with the latest push. 
-If node runs successfully before timeout, it would not show the tooltip.
-Also if node hasIssues will not show the tooltip. 
-Also if Paused node is paused before the tooltip is shown.

And one more scenario. If one of the nodes are paused during execution, the watcher update the prop.